README.md

```diff
@@ -251,21 +251,26 @@ It is recommended to install the latest version of CMake.
 
 #### System Libraries
-The software in the superbuild depends on several libraries such as [Eigen](http://eigen.tuxfamily.org) and [Qt](https://www.qt.io/) that we assume are already available in your machine, as it would be too time intensive to build those libraries in the superbuild itself. 
+The software in the superbuild depends on several libraries such as [Eigen](http://eigen.tuxfamily.org) and [Qt](https://www.qt.io/) that we assume are already available in your machine.
 
-##### YARP, iCub  and robotology-additional-dependencies installers
-The robotology project is currently providing binary installers for YARP and iCub libraries, and one additional installer for additional dependencies.
-As the idea of the superbuild is to easily permit compilation from source of YARP, iCub and all related software, 
-we use these installers only to install the **dependencies** of these libraries, not to install YARP and iCub themselfs. 
-In particular you have to run the following installers:
-* YARP ( http://www.yarp.it/download.html#download_windows ) to install ACE, Eigen3 and Qt5 (disable YARP itself during the installation as it will be installed by the superbuild, and disable OpenCV as the version distributed with YARP  is not working, see https://github.com/robotology/robotology-superbuild/issues/145 for more info)
-* ICUB software ( http://wiki.icub.org/wiki/Windows:_installation_from_sources#Getting_iCub.27s_dependenceis ) to install Ipopt, SDL, Qt5 and GLUT (disable ICUB during the installation, as ICUB will be installed by the superbuild)
-* the robotology-additional-dependencies installer ( https://github.com/robotology-playground/robotology-additional-dependencies ) to install LibXML2,
-* if you need to compile software that depends on OpenCV, download the official installer for OpenCV 3.4.4 at https://sourceforge.net/projects/opencvlibrary/files/3.4.4/opencv-3.4.4-vc14_vc15.exe/download and set the `OpenCV_DIR` enviroment variable to the directory that contains the `OpenCVConfig.cmake` file, tipically `<install_prefix>/opencv/build/x64/vc15/lib/` for 64-bit installations. Note that tipically OpenCV is installed as part of the YARP dependencies installer, but at the moment the YARP dependencies installer has a bug related to OpenCV, see  https://github.com/robotology/robotology-superbuild/issues/145 for more info. Moreover, append `<install_prefix>/opencv/build/x64/vc15/bin/` to the `Path` environment variable.
+To install these libraries on your machine, we suggest to use [`vcpkg`](https://github.com/microsoft/vcpkg), the C++ library manager mantained by Microsoft. As `vcpkg` compiles from sources all its libraries, this can be quite time intensive for some libraries such as `qt5` or `opencv`. 
```

Comment:
The complete list of the libraries is available both in the link https://github.com/robotology-playground/robotology-superbuild-dependencies-vcpkg and in the  later provided link https://github.com/robotology/robotology-superbuild/pull/348/files#diff-5dd97f9483cd18fe5b955d65f1ad5c79 . I am afraid that repeating once more here would be just confusing.

Reply:
Agree on what you said, but I think It is better to add a link for `these` to one of the links you have mentioned.

Reply:
What I am afraid is that if I add a link, the user will follow the link and install vcpkg dependencies manually instead of continue to read, as we had this problem in the past with the `setup.sh` scripts, were we listed env variables before, and **later** we explained that you could load them. What happened with most users is that they were copy-pasting the env variables in their `.bashrc`, without reading further. If it is less confusing, I can change `these libraries` with `the required dependencies`.  cc @diegoferigo

Reply:
I understand your point and it makes sense. Given the fact that at the end of paragraph you mentioned the dependent libraries (`If you prefer to use your own vcpkg to install the dependencies of the superbuild, please refer to the documentation available at [`doc/vcpkg-dependencies.md`](doc/vcpkg-dependencies.md).`), your suggested change should be fine, and there is no need to add a link here.

Reply:
Description addressed in https://github.com/robotology/robotology-superbuild/pull/348/commits/f56c480aa6b068bcbd6b1baa6a0ccd7a48a563f5 .